src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -14,6 +14,7 @@ export const formBenefits = {
   '22-5495': 'education benefits',
   '40-10007': 'pre-need determination of eligibility in a VA national cemetery',
   VIC: 'Veteran ID Card',
+  'complaint-tool': 'feedback',
```

Comment:
We are replacing complain with feedback as it is visible to the user @lihanli @jholton @rianfowler @erikphansen should we plan to replace all instances of "complaint-tool" with "feedback-tool"?